File: Fase 2/ServidorRevistas/src/main/java/objetos/Editor.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package objetos;

/**
 *
 * @author fernanrod
 */
public class Editor {
    private String username;
    private String password;
    private String descripcion;
    private String hobbies;
    private String genero;
    
    public Editor(String username,String password,String descripcion,String hobbies,String genero){
        this.username = username;
        this.password = password;
        this.descripcion = descripcion;
        this.hobbies = hobbies;
        this.genero = genero;
    }
    
    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public String getHobbies() {
        return hobbies;
    }

    public String getGenero() {
        return genero;
    }

}

File: Fase 2/revistas-app/src/app/objetos/base/Editor.ts

import { Etiqueta } from "./Etiqueta";
import { GeneroEnum } from "./GeneroEnum";

export class Editor {
    private _username: string;
    private _password: string;
    private _descripcion: string | null;
    private _hobbies: string | null;
    private _genero: GeneroEnum | null;
    
    constructor(username: string, password: string, descripcion: string, hobbies: string, genero: GeneroEnum){
        this._username = username;
        this._password = password;
        this._descripcion = descripcion;
        this._hobbies = hobbies;
        this._genero = genero;
    }
    
    public get username(): string {
        return this._username;
    }
    public set username(value: string) {
        this._username = value;
    }

    public get password(): string {
        return this._password;
    }
    public set password(value: string) {
        this._password = value;
    }

    public get descripcion(): string|null {
        return this._descripcion;
    }
    public set descripcion(value: string|null) {
        this._descripcion = value;
    }

    public get hobbies(): string|null {
        return this._hobbies;
    }
    public set hobbies(value: string|null) {
        this._hobbies = value;
    }

    public get genero(): GeneroEnum|null {
        return this._genero;
    }
    public set genero(value: GeneroEnum|null) {
        this._genero = value;
    }
}